client/src/components/FeedbackForm/FeedbackForm.tsx

```diff
@@ -39,24 +52,50 @@ const FeedbackForm: FC = () => {
     event: React.FormEvent<HTMLFormElement | HTMLTextAreaElement | HTMLButtonElement>
   ) => {
     event.preventDefault();
-    addFeedback(state);
-    dispatch({ type: FEEDBACK_FORM_RESET });
+    const inputsStateArray = [{ rating }, { nickname }, { email }, { reviewTitle }, { review }];
+    inputsStateArray.forEach((stateObj: object) => {
+      dispatch({
+        type: UPDATE_INPUT_VALUES,
+        payload: { name: getName(stateObj), data: getValue(stateObj) }
```

Comment:
too complex function. you can write the same transformation much shorter using object like this `{ rating, nickname, email, reviewTitle, review }` and `Object.keys` method